Formula/lanraragi.rb

```diff
@@ -99,6 +112,13 @@ def caveats
   end
 
   test do
+    # Make sure lanraragi writes files to a path allowed by the sandbox
+    ENV["LRR_LOG_DIRECTORY"] = ENV["LRR_TEMP_DIRECTORY"] = testpath
+    %w[server.pid shinobu.pid minion.pid].each { |file| touch file }
+
+    # Don't actually run a redis server
+    ENV["LRR_DATABASE_DIRECTORY"] = "probably_not_a_real_directory"
+
     # This can't have its _user-facing_ functionality tested in the `brew test`
     # environment because it needs Redis. It fails spectacularly tho with some
     # table flip emoji. So let's use those to confirm _some_ functionality.
```

Comment:
```suggestion
    # table flip emoji. So let's use those to confirm _some_ functionality.
    # We can't call the "lanraragi" shortcut script as that'll try to kickstart a redis-server, so we have to test
    # by running "npm start" manually.
```
I can't add a suggestion to the `assert_match` line but that's where the fix lies, see the review comment. 🙂